ui-v2/src/components/flow-runs/data-table/duration-cell.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+	type FlowRunWithDeploymentAndFlow,
+	type FlowRunWithFlow,
+} from "@/api/flow-runs";
+import { Icon } from "@/components/ui/icons";
+import { Typography } from "@/components/ui/typography";
+import humanizeDuration from "humanize-duration";
+
+type DurationCellProps = {
+	flowRun: FlowRunWithFlow | FlowRunWithDeploymentAndFlow;
+};
+
+export const DurationCell = ({ flowRun }: DurationCellProps) => {
+	const { estimated_run_time, total_run_time } = flowRun;
+	const duration = estimated_run_time || total_run_time;
+	const durationLabel = humanizeDuration(duration, { maxDecimalPoints: 3 });
```

Comment:
we likely want to customize the options a bit for this. My gut says we want to use "largest: 2" amd 3 decimal places is probably too much. I'm thinking at most 1. Possibly 0. 

Then we might want a tooltip that shows more precision possibly.